benchmarks/base.Dockerfile

```diff
@@ -14,6 +14,8 @@ WORKDIR /app
 RUN apt-get update && apt-get install --no-install-recommends -y \
   curl \
   git \
+  build-essential \
+  uuid-runtime \
```

Comment:
just curious what these were needed for (might just be good to have it on record in case they're not required in the future)

Reply:
```suggestion
  # uuid is used to generate identifier for run if one is not provided
  uuid-runtime \ 
```